From 38b895cddcff51b6cfb47c6b3bbb2044a6d7a6dd Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Mon, 24 Oct 2016 10:08:09 +0200
Subject: [PATCH] impl. missing function

This is copied from vcl/unx/generic/glyphs/gcach_layout.cxx .

Change-Id: I778c1c862c80668a06db8e91c11773ecc9916078
---
 vcl/source/gdi/CommonSalLayout.cxx | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/vcl/source/gdi/CommonSalLayout.cxx b/vcl/source/gdi/CommonSalLayout.cxx
index 4bd9f60..91d7762 100644
--- a/vcl/source/gdi/CommonSalLayout.cxx
+++ b/vcl/source/gdi/CommonSalLayout.cxx
@@ -104,6 +104,15 @@ static void scaleHbFont(hb_font_t* pHbFont, const FontSelectPattern& aFontSelDat
 }
 
 #if !HB_VERSION_ATLEAST(1, 1, 0)
+// Disabled Unicode compatibility decomposition, see fdo#66715
+static unsigned int unicodeDecomposeCompatibility(hb_unicode_funcs_t* /*ufuncs*/,
+                                                  hb_codepoint_t      /*u*/,
+                                                  hb_codepoint_t*     /*decomposed*/,
+                                                  void*               /*user_data*/)
+{
+    return 0;
+}
+
 static hb_unicode_funcs_t* getUnicodeFuncs()
 {
     static hb_unicode_funcs_t* ufuncs = hb_unicode_funcs_create(hb_icu_get_unicode_funcs());
-- 
2.9.3

